applications_tests/lethe-particles/restart_circle_files/restart_circle_0.triangulation.info

```diff
@@ -1,2 +1,2 @@
 version nproc n_attached_fixed_size_objs n_attached_variable_size_objs n_coarse_cells
-5 16 0 1 5
+5 1 0 1 5
```

Comment:
Funny it means the previous restart was made with 16 procs, why? I don't know...
This could explain the results change. Anyway it's good the way you did it.